e2e/coding-interview-prep-intro-page.spec.ts

```diff
@@ -12,15 +13,45 @@ test.describe('Certification intro page', () => {
     await page.close();
   });
 
-  test('Should have a relevant page title', async () => {
+  test('Should have a relevant page title ', async () => {
     await expect(page).toHaveTitle('Coding Interview Prep | freeCodeCamp.org');
   });
 
-  test('Should render', async () => {
-    await expect(
-      page.locator(
-        "text=If you're looking for free coding exercises to prepare for your next job interview, we've got you covered."
-      )
-    ).toBeVisible();
+  test('Should have course heading and relevant icon image in the superblock', async () => {
+    const heading = page.getByTestId('superblock-heading');
+    await expect(heading).toBeVisible();
+    await expect(heading).toContainText('Coding Interview Prep');
+
+    const superblockIcon = page.getByTestId('superblock-icon');
+    await expect(superblockIcon).toBeVisible();
+
+    const algorithmIcon = superblockIcon.getByTestId('algorithm-icon');
+    await expect(algorithmIcon).toBeVisible();
+  });
+
+  test('Should have relevant course description in the superblock', async () => {
+    const description = page.getByTestId('superblock-description-box');
+    const childParagraphs = description.locator('p');
```

Comment:
This can then be moved into a loop structure where each child paragraph can be read.

For example:


```typescript
const description = page.getByTestId('superblock-description-box');
const childParagraphs = description.getByTestId('superblock-description-paras');
await expect(childParagraphs).toHaveCount(2);
for(const para of await childParagraphs.all()){
   await expect(para).toHaveText(...)
}
```

Reply:
Done! @Ahmadkashif Using the test-id to access child paragraphs instead of locator now. 👍🏼

For the looping of array, toEqual is essentially doing the same thing so I'd suggest keeping it the same.